layouts/partials/tw/modal-takeover.html

```diff
@@ -0,0 +1,132 @@
+{{ $imagename := .filename }}
+{{ $hed := .hed }}
+{{ $dek := .dek }}
+{{ $link := .link }}
+
+
+<div x-data x-intersect.once="$dispatch('open-takeover')"></div>
```

Comment:
If it’s being trigger immediately, you can remove this and add an x-intersect.once="show()" to the section and remove the @ listener.

Reply:
Removing line 7 and the @ listener, then adding in `x-intersect.once="show()"` isn't working.